tests/vmi_multus_test.go

```diff
@@ -941,6 +941,61 @@ var _ = Describe("[Serial]SRIOV", func() {
 				})
 			})
 		})
+
+		Context("migration", func() {
+
+			BeforeEach(func() {
+				if err := validateSRIOVSetup(virtClient, sriovResourceName, 2); err != nil {
+					Skip("Migration tests require at least 2 nodes: " + err.Error())
+				}
+			})
+
+			BeforeEach(func() {
+				tests.EnableFeatureGate(virtconfig.SRIOVLiveMigrationGate)
+			})
+
+			AfterEach(func() {
+				tests.DisableFeatureGate(virtconfig.SRIOVLiveMigrationGate)
+			})
+
+			var vmi *v1.VirtualMachineInstance
+			var interfaceName string
+
+			const mac = "de:ad:00:00:be:ef"
+
+			BeforeEach(func() {
+				// The SR-IOV VF MAC should be preserved on migration, therefore explicitly specify it.
+				vmi = getSriovVmi([]string{sriovnet1}, defaultCloudInitNetworkData())
+				vmi.Spec.Domain.Devices.Interfaces[1].MacAddress = mac
+
+				vmi = startVmi(vmi)
+				vmi = waitVmi(vmi)
+
+				Eventually(func() error {
```

Comment:
Can you please add a comment explaining why `Eventually` is needed?

Reply:
Done.
Also reduced the timeout. Once the guest-agent is up, it should not take more than 30sec to get the interface list.